### DISCUSSION TITLE
No dryer

### DISCUSSION THREAD

**Discussion by timewolf** [ekutner/home-connect-hass#443] [Community Support]
hi,
i used home connect alt with my fridge, recently i bought a dryer, added to bosh home connect, but I cannot find it on HA.
I removed the integration, reloaded, added again credential, but no dryer, only the fridge.
The dryder is online and connected via bosh app.
any ideas?

**Comment by ekutner** [comment]
Review the troubleshooting and FAQ in the README

**Comment by timewolf** [comment]
The appliance is powered on and connected to the Home Connect service 

Wifi phone is OFF, i can see the dryer into the Home Connect APP.

I've removed power cord and insert back, turned on. I still can see the Dryer into Home Connect, neither in HA Home Connect ALT

**Reply by ekutner** [reply]
It's very unlikely, because when the wifif is turned off the app is using the same API as the integration, so if it sees the dryer the integration should see it too but to verify please open an issue and attach the debug log (make sure to set log level to 7).
